### PR TITLE
Sonobi: Added consent macros to the user sync url

### PR DESCRIPTION
**Pull request by JonGoSonobi**
Updated Sonobi's user sync endpoint to fill the gdpr and us privacy query params

### CODE REVIEW

File: static/bidder-info/sonobi.yaml [hunk 1]
```diff
@@ -13,5 +13,5 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://sync.go.sonobi.com/us.gif?loc={{.RedirectURL}}"
+    url: "https://sync.go.sonobi.com/us.gif?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&loc={{.RedirectURL}}"
```
[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-info/sonobi.yaml:16]: @JonGoSonobi  could you address above comment

[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-info/sonobi.yaml:16]: @JonGoSonobi ping to address above PR comments

**Comment by gargcreation1992** [comment]
@JonGoSonobi Requesting you to please address the open comment.

**Comment by onkarvhanumante** [comment]
@JonGoSonobi closing this PR due to inactivity. PR can be reopened when new changes are pushed on sonobi:sonobi-add-consent-to-usersyncs

**Comment by JonGoSonobi** [comment]
Hey @onkarvhanumante, I reviewed this over with the team. The sync urls that fail to redirect is because the gdpr_consent string provided does not define that Sonobi is consented to user snyc. So, the endpoint will not redirect to the &loc query parameter.

Should our user sync endpoint redirect even if the gdpr_consent string doesnt say Sonobi is consented?

**Comment by JonGoSonobi** [comment]
@onkarvhanumante any chance you were able to look at my last comment? I'll re-open this PR if this thread has gone stale.

**Pull request closed without merging**